nmdc_runtime/site/translation/gold_translator.py

```diff
@@ -210,11 +210,24 @@ def _get_quantity_value(
             return None
 
         numeric_value = None
+        has_minimum_numeric_value = None
+        has_maximum_numeric_value = None
+
         # TODO: in the future we will need better handling
         # to parse out the numerical portion of the quantity value
         # ex. temp might be 3 C, and we will need to parse out 3.0 from it
         if unit == "meters":
             numeric_value = nmdc.Double(field_value)
+            
+            has_maximum_numeric_value = gold_entity.get("depthInMeters2")
```

Comment:
`_get_quantity_value` is supposed to be a generic method, so I don't like hardcoding `depthInMeters2` here. It also doesn't really make a lot of sense to me why we only possibly set `has_minimum_numeric_value` and `has_maximum_numeric_value` based on the `unit`?? Why not add parameters for `minimum_value_field` and `maximum_value_field`?

Reply:
Yup, I was contemplating the specificity of this solution too 😫 I'll make the logic more generic keeping your suggestions in mind, and add another commit to this PR!

If not accepting `minimum_value_field` and `maximum_value_field` as arguments of `_get_quantity_value()` we can also allow the `gold_field` argument to accept `Union[str, List[str]]` and if it gets a list of two items, then use the first value/field in that list to get the minimum value, and the second value/field in that list to get the maximum value.

Reply:
That makes sense too! Although if you go the second route use `Tuple[str, str]` not `List[str]` to indicate that it's not an arbitrary length list, but rather a pair of strings.

Reply:
That's a great suggestion! I used `Tuple[str, str]`.